inst/touchstone-comment.yaml

```diff
@@ -15,38 +15,4 @@ jobs:
       ${{ github.event.workflow_run.event == 'pull_request' &&
       github.event.workflow_run.conclusion == 'success' }}
     steps:
-      - name: 'Download artifact'
-        uses: actions/github-script@v3.1.0
-        with:
-          script: |
-            var artifacts = await github.actions.listWorkflowRunArtifacts({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               run_id: ${{github.event.workflow_run.id }},
-            });
-            var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
-              return artifact.name == "pr"
-            })[0];
-            var download = await github.actions.downloadArtifact({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               artifact_id: matchArtifact.id,
-               archive_format: 'zip',
-            });
-            var fs = require('fs');
-            fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
-      - run: unzip pr.zip
-      - name: 'Comment on PR'
-        uses: actions/github-script@v3
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            var fs = require('fs');
-            var issue_number = Number(fs.readFileSync('./NR'));
-            var body = fs.readFileSync('./info.txt').toString();
-            await github.issues.createComment({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              issue_number: issue_number,
-              body: body
-            });
+      - uses: lorenzwalthert/touchstone/actions/comment@standalone-action
```

Comment:
That way, I am not sure we can't version the action can we? Wouldn't it make more sense like this?
```
lorenzwalthert/touchstone/actions/comment@main
```

Reply:
ah well this was mainly for testing (as the actions are not on main yet), we could then create a tag so people can use a fixed version of the action

Reply:
ah ok I see. Won't have time now, but will test it tomorrow.

Reply:
(I first did not get that the part after `@` is the branch)